controllers/serverTableController.js

```diff
@@ -1,27 +1,65 @@
-var app = angular.module('pokemonDraft', []);
-app.controller('serverTableController', function($scope, $http) {
+var app = angular.module('pokemonDraft', ['ui.bootstrap.modal', 'ui.bootstrap.tpls']);
```

Comment:
var is not used that much anymore, due to es6 becoming main stream. Have a google for angular transpiling to es6 and let me know if it's main stream at all, as es6 is by far main stream at the moment. (ie variables are declared using const and let, which restrict them to the current block scope and don't polute the global namespace like var does